src/index.js

```diff
@@ -109,6 +131,8 @@ function processDdoc(ddoc) {
       .then(() => apps.stop())
       .then(() => info('All apps stopped.'))
 
+      .then(() => deployDdoc(ddoc))
```

Comment:
Not sure what you mean sorry. There is a logging statement at the start of the `deployDdoc` function, unless I misunderstand your question.